src/In.ProjectEKA.HipService/DataFlow/MessagingQueueListener.cs

```diff
@@ -0,0 +1,65 @@
+
+namespace In.ProjectEKA.HipService.DataFlow
+{
+    using System;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.ObjectPool;
+    using Newtonsoft.Json;
+    using RabbitMQ.Client;
+    using RabbitMQ.Client.Events;
+    
+    internal class MessagingQueueListener : BackgroundService
+    {
+        private readonly DefaultObjectPool<IModel> objectPool;
+        private IModel channel;
+        
+        public MessagingQueueListener(IPooledObjectPolicy<IModel> objectPolicy)  
+        {  
+            objectPool = new DefaultObjectPool<IModel>(objectPolicy, Environment.ProcessorCount * 2);
+            SetupMessagingQueue();
+        }
+
+        private void SetupMessagingQueue()
+        {
+            channel = objectPool.Get();
+            channel.ExchangeDeclare(MessagingQueueConstants.DataRequestExchangeName, ExchangeType.Topic, true);  
+            channel.QueueDeclare(
+                MessagingQueueConstants.DataRequestRoutingKey,
+                true,
+                true,
+                true,
+                null);  
+            channel.QueueBind(
+                MessagingQueueConstants.DataRequestRoutingKey,
+                MessagingQueueConstants.DataRequestExchangeName, 
+                MessagingQueueConstants.DataRequestRoutingKey, 
+                null);  
+            channel.BasicQos(0, 1, false);     
+        }
+        
+        protected override Task ExecuteAsync(CancellationToken stoppingToken)
+        {
+            stoppingToken.ThrowIfCancellationRequested();
+            var consumer = new EventingBasicConsumer(channel);
+            consumer.Received += (ch, ea) =>
+            {
+                Console.Write("[*] Received Message");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4963271)